src/main/java/org/opentripplanner/standalone/config/routerconfig/TransitRoutingConfig.java

```diff
@@ -200,6 +202,16 @@ public TransitRoutingConfig(
         )
         .asDurations(PAGING_SEARCH_WINDOW_ADJUSTMENTS);
 
+    this.maxSearchWindow =
+      c
+        .of("maxSearchWindow")
+        .since(V2_4)
+        .summary("Upper limit of the request parameter searchWindow.")
+        .description(
+          "Maximum search window that can be set through the searchWindow API parameter."
+        )
+        .asDuration(Duration.ofHours(24));
```

Comment:
The default value should not be defined here - not the responsibility of this class. Can you remind me that we fix this after this is merged? This follows an existing pattern and I would like to fix all of them - and that does not belong in this PR.